### PR TITLE
Added sleep measures API and incorporated in demo app

### PR DESCRIPTION
**Pull request by robertturrall**
Retrieval of sleep measures and inclusion in the demo App.

### CODE REVIEW

File: Withings-SDK-iOS/WithingsMeasureAPIClient.m [hunk 1]
```diff
@@ -234,8 +261,18 @@ + (MeasuresDateRange*)dateRangeBetweenStartDate:(NSDate*)startDate andEndDate:(N
 {
     NSDate *startDate = _startDate ? _startDate : [NSDate dateWithTimeIntervalSince1970:0];
     NSDate *endDate = _endDate ? _endDate : [NSDate date];
-    return @{@"startdate" : @([startDate timeIntervalSince1970]),
```
Comment (jdrevet) [inline Withings-SDK-iOS/WithingsMeasureAPIClient.m:237]: @robertturrall Could you explain why you need to transform the date sent to the server ?

Comment (robertturrall) [inline Withings-SDK-iOS/WithingsMeasureAPIClient.m:237]: @jdrevet This is to satisfy the requirements specified in the Withings API documentation: http://oauth.withings.com/api/doc#api-Measure-get_sleep - "(recommended value : set to Noon in the user time zone)"

Comment (jdrevet) [inline Withings-SDK-iOS/WithingsMeasureAPIClient.m:237]: @robertturrall Ok. The problem is that the parameters method of MeasuresDateRange is also used for body measures. So, could you create a specific method to get the datetime related to noon for sleep API only ?

Comment (robertturrall) [inline Withings-SDK-iOS/WithingsMeasureAPIClient.m:237]: @jdrevet Of course. I had thought that I had done this but it seems not. I'll do that and also check everything through again. Apologies for the oversight.

Comment (robertturrall) [inline Withings-SDK-iOS/WithingsMeasureAPIClient.m:237]: Done and pushed

**Pull request merged**